explorer/explorer_test.go

```diff
@@ -525,9 +529,23 @@ func TestExplorerApi(t *testing.T) {
 	require.Nil(res.Block)
 	require.Nil(res.Transfer)
 	require.Nil(res.Vote)
+	require.Nil(res.Address)
 	require.Equal(&executions[0], res.Execution)
 	require.Equal(len(executions), 2)
 
+	transfers, err = svc.GetLastTransfersByRange(4, 1, 3, true)
+	require.NoError(err)
+	res, err = svc.GetBlockOrActionByHash(transfers[0].Sender)
```

Comment:
Since we deprecated source address in all actions, transfers[0].Sender is always an empty string, which results in a non-existing account issue. Actually, you don't need to get transfers first to test the new function. You can just use an existing account, e.g. ta.Addrinfo["charlie"], as the argument for GetBlockOrActionByHash.